### PR TITLE
fix(logo): change footer logo to use svg

### PR DESCRIPTION
**Pull request by ddonahue007**
Screen Shot 2020-03-25 at 9 21 37 AM

### CODE REVIEW

File: packages/gatsby-theme-patternfly-org/components/footer/footer.js [hunk 1]
```diff
@@ -156,11 +155,7 @@ export class Footer extends React.Component {
...
...BaIE0yMC4zMDk1ODksMjMuNDY0OCBMMjQuMTE4MDM2NSwxOC4yNjQ4IEwyNC43MTQxNTUzLDE5LjYxODQgTDIxLjUxMDYxNjQsMjQuMTI4IEwyMC4wNzE0NjEyLDIzLjc5NjggTDIwLjMxMTk4NjMsMjMuNDY2NCBMMjAuMzA5NTg5LDIzLjQ2NDggWiBNMTkuNjIyMzc0NCwyMy4yNzY4IEwxNy40ODAwMjI4LDMuMTc5MiBMMjMuODA3OTkwOSwxNy41NjA4IEwxOS42MjIzNzQ0LDIzLjI3NjggWiBNMTcuOTg1MDQ1NywyLjY3NjggTDI2LjgwMjE2ODksMTMuNDcyIEwyNC4yNjI2NzEyLDE2Ljk0IEwxNy45ODUwNDU3LDIuNjc2OCBaIE02Ljk2NjQzODM2LDI1LjI4NTYgTDAuODcwMjA1NDc5LDE2LjY0MDggTDEyLjc5ODk3MjYsNC45NjcyIEw1Ljg3MDg5MDQxLDEzLjQ1MDQgTDguOTg0MTMyNDIsMTcuNzAwOCBMOC4xMDU5MzYwNywxOS42OTYgTDExLjM1OTgxNzQsMjQuMjc2OCBMNi45NjY0MzgzNiwyNS4yODU2IFogTTE2Ljc5MDQxMSwzMC43NzEyIEwxMi40Nzg1Mzg4LDI0LjcwMzIgTDEzLjg3MDU0NzksMjQuMzgyNCBMMTYuNzUxMjU1NywyOC4zNDk2IEwxOS42NDE1NTI1LDI0LjM4MjQgTDIxLjA5MTA5NTksMjQuNzE2OCBMMTYuNzkwNDExLDMwLjc3MTIgWiBNMjIuMjExNDE1NSwyNC4yODk2IEwyNS40NzQwODY4LDE5LjY5NiBMMjQuNTcxOTE3OCwxNy42NDQ4IEwyNy42NDM2MDczLDEzLjQ1MDQgTDIwLjcxNTUyNTEsNC45NjcyIEwzMi42NDM0OTMyLDE2LjY0MDggTDI2LjU0ODA1OTQsMjUuMjg2NCBMMjIuMjExNDE1NSwyNC4yODk2IFoiIGlkPSJTaGFwZSIgZmlsbD0idXJsKCNsaW5lYXJHcmFkaWVudC0xKSI+PC9wYXRoPgogICAgICAgICAgICAgICAgPC9nPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgPC9nPgo8L3N2Zz4=" alt="Patternfly Logo" /></a>
```
Comment (dlabrecq) [inline packages/gatsby-theme-patternfly-org/components/footer/footer.js]: In trainingLayout.js and sideNavLayout.js, we're using the Brand component, which will eliminate the need for the CSS selectors. We shouldn't need to embed the SVG, but here is an example.

https://github.com/patternfly/patternfly-org/blob/master/packages/gatsby-theme-patternfly-org/layouts/trainingLayout/trainingLayout.js#L40

File: packages/gatsby-theme-patternfly-org/components/footer/footer.js [hunk 2]
```diff
@@ -156,11 +157,7 @@ export class Footer extends React.Component {
             <GridItem sm={12} md={4}>
               <Grid className="pf-u-pt-xl ws-org-pfsite-l-footer-column">
                 <GridItem className="pf-u-px-xl">
-                  <img
-                    src={patternflyLogo}
-                    alt="PatternFly logo"
-                    className="ws-org-pfsite-footer-menu-about-logo pf-u-pb-md"
-                  />
+                  <a class="pf-c-page__header-brand-link pf-c-brand ws-org-pfsite-footer-menu-about-logo pf-u-pb-md" href="/"><Brand src={logo} alt="Patternfly Logo" /></a>
```
Comment (dlabrecq) [inline packages/gatsby-theme-patternfly-org/components/footer/footer.js]: It appears that we already have this logo in packages/gatsby-theme-patternfly-org/layouts/logo.svg?

File: packages/gatsby-theme-patternfly-org/components/footer/footer.js (hunk 1 above)
Comment (ddonahue007) [reply packages/gatsby-theme-patternfly-org/components/footer/footer.js]: Fixed

File: packages/gatsby-theme-patternfly-org/components/footer/footer.js (hunk 2 above)
Comment (ddonahue007) [reply packages/gatsby-theme-patternfly-org/components/footer/footer.js]: Fixed

**Pull request merged**